src/python/content.cpp

```diff
@@ -1740,13 +1740,17 @@ parameters2dict(const ak::util::Parameters& in) {
   for (auto pair : in) {
     std::string cppkey = pair.first;
     std::string cppvalue = pair.second;
-    py::str pykey(PyUnicode_DecodeUTF8(cppkey.data(),
-                                       cppkey.length(),
-                                       "surrogateescape"));
-    py::str pyvalue(PyUnicode_DecodeUTF8(cppvalue.data(),
-                                         cppvalue.length(),
-                                         "surrogateescape"));
+    PyObject* tmp1 = PyUnicode_DecodeUTF8(cppkey.data(),
+                                          cppkey.length(),
+                                          "surrogateescape");
+    PyObject* tmp2 = PyUnicode_DecodeUTF8(cppvalue.data(),
+                                          cppvalue.length(),
+                                          "surrogateescape");
+    py::str pykey(tmp1);
+    py::str pyvalue(tmp2);
```

Comment:
```suggestion
    py::str pykey = reinterpret_steal<py::str>(PyUnicode_DecodeUTF8(cppkey.data(),
                                          cppkey.length(),
                                          "surrogateescape"));
    py::str pyvalue = reinterpret_steal<py::str>(PyUnicode_DecodeUTF8(cppvalue.data(),
                                          cppvalue.length(),
                                          "surrogateescape"));
```

Reply:
I think we can do this using `pybind11`: https://pybind11.readthedocs.io/en/stable/reference.html#_CPPv4I0E18reinterpret_borrow1T6handle

Reply:
(And repeat for all instances)